it.mijorus.collector.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "it.mijorus.collector",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "command": "collector",
+    "finish-args": [
+        "--filesystem=home",
```

Comment:
Does this use https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.FileTransfer.html ?

Reply:
I'm using Gtk's built in DropAreaTarget component, which I guess is using the portal, but I'm not sure.

Btw, when files are dropped into the window, they are not copied or anything, it would just be a waste of time and disk space.

The app  only keeps a reference path

Reply:
> Btw, when files are dropped into the window, they are not copied or loaded in memory, it would just be a waste of time and disk space.

The file portal doesn't copy anything.

Reply:
I need to test the app without this permission.

Reply:
Doesn't seem to work without the filesystem permission

Reply:
I'll see if I can make the app working without this permission